modules/letsencrypt/controllers/clusterdomain/controller.go

```diff
@@ -66,6 +66,7 @@ func (f *certsHandler) onCertChange(key string, obj *certmanagerv1alpha2.Certifi
 func (f *certsHandler) Handle(obj *v1.ClusterDomain, status v1.ClusterDomainStatus) ([]runtime.Object, v1.ClusterDomainStatus, error) {
 	if obj.Spec.SecretName != "" {
 		status.AssignedSecretName = obj.Spec.SecretName
+		status.HTTPSSupported = true
```

Comment:
Why do we need to set this? Shouldn't `status.HTTPSSupported = condition.Cond("Ready").IsTrue(cert)` on line 85 cause this to switch automatically?

Reply:
If user specify secretName we will not request letsencrypt certs so we can't use that cert condition to set `HTTPSSupported`. This means user is managing secret by their side.